handle_hosts.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/olekukonko/tablewriter"
+	karma "github.com/reconquest/karma-go"
+)
+
+func handleHosts(
+	zabbix *Zabbix,
+	config *Config,
+	args map[string]interface{},
+) error {
+	var (
+		hostnames, _  = parseSearchQuery(args["<pattern>"].([]string))
+		removeHost, _ = args["--remove"].(string)
+
+		err               error
+		hostsTable, hosts []Host
+	)
+
+	destiny := karma.Describe("method", "handleHosts")
+
+	switch {
+	case removeHost != "":
+
+		err = handleRemoveHosts(zabbix, config, args)
+		if err != nil {
+			return destiny.Describe(
+				"error", err,
+			).Describe(
+				"hostname", removeHost,
+			).Reason(
+				"can't remove zabbix hosts",
+			)
+		}
+
+	default:
+
+		for _, hostname := range hostnames {
+			hosts, err = searchHosts(zabbix, hostname)
+			if err != nil {
+				return destiny.Describe(
+					"error", err,
+				).Describe(
+					"hostname", hostname,
+				).Reason(
+					"can't search zabbix hosts",
+				)
+			}
+			hostsTable = append(hostsTable, hosts...)
+
+		}
+		if len(hostsTable) > 0 {
+			printHostsTable(hostsTable)
+		}
+
+	}
+	return nil
+}
+
+func handleRemoveHosts(
+	zabbix *Zabbix,
+	config *Config,
+	args map[string]interface{},
+) error {
+	var (
+		removeHost, _ = args["--remove"].(string)
```

Comment:
What is `removeHost`? It's a pattern or a single hostname?

Reply:
It's single hostname, this is checked after search host and return error https://github.com/zarplata/zabbixctl/blob/maintenance%2Bhost%2Btest/handle_hosts.go#L93